inflate.c

```diff
@@ -1112,14 +1112,18 @@ int32_t Z_EXPORT PREFIX(inflate)(PREFIX3(stream) *strm, int32_t flush) {
      */
   inf_leave:
     RESTORE();
+    uint32_t checkBytes = out - strm->avail_out;
```

Comment:
I think maybe you meant to do snakecase instead of camelcase.

Reply:
ah, sorry camel is my personal preference that's hard to break. Fixing now.

Reply:
I use it all the time too because like 70% of what I do is in JavaScript these days.